test/spec.index.js

```diff
@@ -891,10 +904,14 @@ describe('Template Mixins', function () {
                 }, { translate: translate });
                 middleware(req, res, next);
                 res.locals['select']().call(res.locals, 'field-name');
-                render.should.have.been.calledWith(sinon.match({
-                    options: [
-                        { label: '', selected: false, toggle: undefined, value: '' }
-                    ]
+                render.should.have.been.calledWith(sinon.match(function (value) {
+                    var obj = value.options[0]
+                    return _.isMatch(obj, {
```

Comment:
`_.isMatch` will check the given key/value pairs match, but sinon.match, passing an object, checks the object is identical. With using the custom matcher there can be extra keys that aren't specified such as functions